utils/formUtils.ts

```diff
@@ -24,6 +27,8 @@ export function createStep({
   type: StepType
   section: string
   render: Function
+  renderBasic: Function
```

Comment:
Make this optional and default to the value of `render` if it's not set?

Reply:
I've just been changing it so that it returns a function with a react fragment whenever a new step is called as that's how steps without a "render" call function - should both be optional?